code/modules/spells/targeted/frenchcurse.dm

```diff
@@ -0,0 +1,39 @@
+/spell/targeted/frenchcurse
+	name = "French Curse"
+	desc = "This curse will silence your target for a very long time."
+
+	school = "evocation"
+	charge_max = 600
```

Comment:
Specifically you can use acid to get around the horsemask curse. Here the clothing is unacidable. I can reduce it if you think it would be better to do so though.